dist-persist/wbstack/src/Internal/WbStackPlatformReservedUser.php

```diff
@@ -107,7 +107,7 @@ public static function createOauthConsumer($consumerName, $version, $grants, $ca
         return true;
     }
 
-    public static function getOAuthConsumer($consumerName, $version) {
+    public static function getOAuthConsumer($consumerName, $version, $includeAccess = false) {
```

Comment:
I was thinking that it could maybe sense to refactor the name of this method to try and make it clear to future callers what the purpose is:

usecase 1 with `$includeAccess = false` is: I want an OAuth consumer that each individual user can authorise to edit on their behalf with their account

usecase 2 with  `$includeAccess = true` is: I want credentials to edit the wiki as the PlatformReservedUser

I almost wonder if this would be better as a totally separate function. Really the common logic is more "incidental" than "conceptual"; it just happens to be that to get the credentials for usecase 2 you have to get a consumer but really you aren't asking for a consumer but instead editing credentials.